spec/pdfs/distribution_pdf_spec.rb

```diff
@@ -1,72 +1,91 @@
-# avoid Rubocop failing with an infinite loop when it checks this cop
-# rubocop:disable Layout/ArrayAlignment
+require_relative("../support/distribution_pdf_helper")
+require_relative("../../lib/test_helpers/pdf_comparison_test_factory")
+
+RSpec.configure do |c|
+  c.include DistributionPDFHelper
+  c.include PDFComparisonTestFactory
```

Comment:
These shouldn't need to be included in the base RSpec config. I generally prefer moving everything to module/class methods instead of instance methods. There aren't any instance variables, so doing that makes everything way easier to work with - so you can just call `PDFComparisonTestFactory.create_comparison_pdfs` directly, and don't have to pollute existing specs or configs.